src/script/guards/Message.ts

```diff
@@ -29,3 +29,6 @@ export const isContentMessage = (
 
 export const isMemberMessage = (message: any | undefined | null): message is MemberMessage =>
   message && 'super_type' in message && message.super_type === SuperType.MEMBER;
+
+export const isMessage = (message: any | undefined | null): message is Message =>
```

Comment:
We can simplify it to just `any` then, it gives the same result, as `undefined | null` and everything else is a subset of any